force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -270,12 +340,20 @@ public with sharing class FlowControlLoopsCollections {
      * @return The position of the word in the list as a string or -1 if the word was not found.
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
+        
         // The variable to store the index of the word
 
         // Loop through the list of words
+        for(Integer i = 0; i < words.size(); i++){
+            if(words.get(i) == wordToFind){
+                return i;
```

Comment:
This is a good way to return the position. Can you try another way that would save to a variable and then exit the loop?